content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-lighttpd.adoc

```diff
@@ -0,0 +1,103 @@
+---
+layout: subsection
+---
+
+ifdef::backend-html5[]
+ifndef::env-github[:imagesdir: ../../../resources/managing]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+[[running-jenkins-behind-lighttpd]]
+= Reverse proxy - Lighttpd
+
+In situations where you have existing web sites on your server, you may
+find it useful to run Jenkins (or the servlet container that Jenkins
+runs in) behind https://www.lighttpd.net/[Lighttpd], so that you can bind Jenkins
+to the part of a bigger website that you may have or just centralize the TLS
+termination in a single place.
+This section discusses some of the approaches for doing this.
+
+The https://redmine.lighttpd.net/projects/lighttpd/wiki/Mod_proxy[mod_proxy] works
+by making Lighttpd perform as a "reverse proxy" — when a request arrives for
+certain URLs, Lighttpd becomes a proxy and forwards that request to
+Jenkins, then forwards the response from Jenkins back to the client. 
+
+There are several different alternatives to configure Jenkins with Lighttpd.
+Choose the technique that best meets your needs:
+
+* <<By host>>
+* <<By path>>
+
+By https://www.lighttpd.net/2018/11/28/1.4.52/[default] Lighttpd will force
+the URL normalization, mismatching the headers used by internal Jenkins reverse
+proxy test, showing the message *It appears that your reverse proxy setup is broken*
+on the manage page.
+To avoid this, explicit disable the option `url-path-2f-decode`.
+
+== By host
+
+In the configuration below there is no context path for Jenkins URL.
+
+When a request for the domain `jenkins.example.com` arrives, Lighttpd
+proxy this request to Jenkins.
```

Comment:
```suggestion
When a request for the domain `jenkins.example.com` arrives, Lighttpd proxies this request to Jenkins.
```
This seems like an area where it should be performing the proxy, but the way it is written does not necessarily convey that. Either making it an action verb or adding a word between Lighttpd and proxy.

Reply:
I added the "proxies" verb in the original and will reject only this.